ui/src/app/settings/actions/config.js

```diff
@@ -13,4 +13,25 @@ config.fetch = () => {
   };
 };
 
+config.update = params => {
+  const configParams = Object.keys(params).map(key => {
```

Comment:
Hmm I don't think we should be doing any data manipulation here. Generally redux actions are supposed to be dumb, i.e. take a payload => put it in an action object. So I would map the params object before it gets dispatched.